circleci/images/citusupgradetester/Dockerfile

```diff
@@ -51,7 +51,9 @@ RUN echo "deb https://apt.postgresql.org/pub/repos/apt bullseye-pgdg main" >> /e
  && apt-get update \
 # infer the pgdgversion of postgres based on the $PG_VERSION
  && pgdg_version=$(apt list -a postgresql-server-dev-${PG_MAJOR} 2>/dev/null | grep "${PG_VERSION}" | awk '{print $2}' | head -n1 ) \
- && apt-get install -y --no-install-recommends \
+ && apt-get install -y --no-install-recommends --allow-downgrades \
```

Comment:
we add `--allow-downgrades` as it seems that libpq5/libpq-dev is already included before this step. Probably in the base image, given there is no installed package that depends on either of these packages.

Since the base image might upgrade the version to a newer version we want to be testing against the installation of the packages could force a downgrade, which would cause an error without this flag.